python/tk_multi_publish2/progress/progress_details_widget.py

```diff
@@ -98,6 +98,19 @@ def show(self):
         self.__recompute_position()
         self.ui.log_tree.expandAll()
 
+        # Scroll to last item by default
```

Comment:
Why do we need to scroll to the last item?

Reply:
By default, opening the logs will always show the first log entry/top of the log, which is normally not where errors or the latest status message is at.

This removes the tedious, common artist routine of scrolling to the bottom of the log to figure out what happened